src/Database/DBStructure.php

```diff
@@ -1062,6 +1070,24 @@ public static function checkInitialValues()
 			}		
 		}
 
+		if (self::existsTable('user') && DBA::exists('user', ['uid' => 0])) {
```

Comment:
Why not combine this with lines 1057-1063?

Reply:
It's a fight with the foreign keys

Reply:
Got it